### PR TITLE
`azurerm_orchestrated_virtual_machine_scale_set` - support for the `user_data_base64` property

### PR DESCRIPTION
**Pull request by gcampbell12**
Adds user data to VMSS azurerm_orchestrated_virtual_machine_scale_set resource, the VMSS API requires this value to be base64 encoded hence the schema validation

### CODE REVIEW

**Comment by gcampbell12** [comment]
Acceptance test results:
```bash
==> Checking that code complies with gofmt requirements...
==> Checking that Custom Timeouts are used...
==> Checking that acceptance test packages are used...
TF_ACC=1 go test -v ./internal/services/compute -run=TestAccOrchestratedVirtualMachineScaleSet_otherUserData -timeout 60m -ldflags="-X=github.com/hashicorp/terraform-provider-azurerm/version.ProviderVersion=acc"
=== RUN   TestAccOrchestratedVirtualMachineScaleSet_otherUserData
=== PAUSE TestAccOrchestratedVirtualMachineScaleSet_otherUserData
=== CONT  TestAccOrchestratedVirtualMachineScaleSet_otherUserData
--- PASS: TestAccOrchestratedVirtualMachineScaleSet_otherUserData (460.98s)
PASS
ok      github.com/hashicorp/terraform-provider-azurerm/internal/services/compute       462.354s
```

File: website/docs/r/orchestrated_virtual_machine_scale_set.html.markdown [hunk 1]
```diff
@@ -101,6 +101,8 @@ resource "azurerm_orchestrated_virtual_machine_scale_set" "example" {
 
 * `termination_notification` - (Optional) A `termination_notification` block as defined below.
 
+* `user_data` - (Optional) The Base64-Encoded User Data which should be used for this Virtual Machine Scale Set.
```
Comment (katbyte) [inline website/docs/r/orchestrated_virtual_machine_scale_set.html.markdown:104]:
```suggestion
* `user_data_base64` - (Optional) The Base64-Encoded User Data which should be used for this Virtual Machine Scale Set.
```

**Review by katbyte — CHANGES_REQUESTED** [review]
Thanks @gcampbell12, this LGTM thou i wonder if we should be explicit in what the property expects and call it `user_data_base64` as we often put json etc on the end to make it clear

File: website/docs/r/orchestrated_virtual_machine_scale_set.html.markdown (hunk 1 above)
Comment (gcampbell12) [reply website/docs/r/orchestrated_virtual_machine_scale_set.html.markdown]: Have updated the property to `user_data_base64` as suggested, I originally mirrored the linux VMSS but this makes sense to me

**Review by katbyte — APPROVED** [review]
Thanks @gcampbell12 - LGTM now ⚙️

**Pull request merged**